client/src/Tools/_framework/Paths/Curation.tsx

```diff
@@ -0,0 +1,664 @@
+import {
```

Comment:
Eventually, we should combine this with Activities. However, since Activities is currently in the middle of significant changes, we should hold off on that until the changesets are merged.